### PR TITLE
Initial version of ralph-cli skeleton.

### PR DESCRIPTION
**Pull request by xor-xor**
This PR contains all the data types and functions identified so far, which should give an idea how this application is designed. It should be considered as a "pre-MVP" stage, so although it compiles successfully, it doesn't do anything useful yet.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
 
-build: deps golang-crosscompile
-	source golang-crosscompile/crosscompile.bash; \
-	go-darwin-amd64 build -o dist/ralph-scan-Darwin-x86_64; \
-	go-linux-386 build -o dist/ralph-scan-Linux-i386; \
-	go-linux-amd64 build -o dist/ralph-scan-Linux-x86_64; \
-	go-windows-386 build -o dist/ralph-scan.exe
+build-all: deps
+	@echo "Building ralph-cli binaries for Darwin/Linux/Windows (64-bit)..."
+	env GOOS=darwin GOARCH=amd64 go build $(PACKAGE_NAME) -o dist/ralph-cli-Darwin-x86_64
```
Comment (mkurek) [inline Makefile]: this will not work with bash

Comment (xor-xor) [inline Makefile]: Why not..? It certailny works for me, and I'm using `bash`, not `fish`. Although I've just notice one error here (not related to any shell, though): the `-o` option should go before `$(PACKAGE_NAME)`, so it should be like this:
```
build-all: deps
	@echo "Building ralph-cli binaries for Darwin/Linux/Windows (64-bit)..."
	env GOOS=darwin GOARCH=amd64 go build -o dist/ralph-cli-Darwin-x86_64 $(PACKAGE_NAME)
	env GOOS=linux GOARCH=amd64 go build -o dist/ralph-cli-Linux-x86_64 $(PACKAGE_NAME)
	env GOOS=windows GOARCH=amd64 go build -o dist/ralph-cli.exe $(PACKAGE_NAME)
```
(and the above certainly works with `bash` - I've just checked)

Comment (mkurek) [inline Makefile]: ok - so ignore my comment?

**Comment by mkurek** [comment]
:ok:

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+language: go
+
+go:
+  - 1.6
+  - tip
+
+before_install:
+  - go get github.com/Masterminds/glide
+  - go get golang.org/x/tools/cmd/cover
+  - go get github.com/golang/lint/golint
```
Comment (mkurek) [inline .travis.yml]: these packages will be useful for local development/testing as well - maybe we could "hide" them in Makefile (ex. `make install-test`)?

Comment (xor-xor) [inline .travis.yml]: Yes, probably. But at this point I wouldn't bother with that (we don't have any tests yet anyway).

**Pull request merged**